awless/README.md

```diff
@@ -7,6 +7,13 @@ tagline: |
 
 To update or switch versions, run `webi awless@stable` (or `@v2`, `@beta`, etc).
 
+
+### Files
+
+```text
+~/.local/bin/awless
+~/.local/opt/awless-{version}
+```
```

Comment:
I believe this also has config files, just like flutter and dotenv-linter. See those notes and check the official docs, please.

Reply:
I've added the aws credentials location, that's the only required config file.